kubernetes/loculus/values.schema.json

```diff
@@ -1144,6 +1158,21 @@
       "required": ["enabled", "urls"],
       "additionalProperties": false
     },
+    "fileSharing": {
+      "type": "object",
+      "groups": ["general"],
+      "description": "Settings related to the file sharing feature.",
+      "properties": {
+        "outputFileUrlType": {
+          "type": "string",
+          "groups": ["general"],
+          "enum": ["website", "backend", "s3"],
+          "description": "Whether the backend should supply file URLs as links using the website, or directly to the S3 objects."
```

Comment:
What's the meaning of the enum value backend? I don't see it described in description in contrast to website and s3

Reply:
Would be good to add to `description`

Reply:
The ideal place to make such suggestions is in an issue or a PR :)